### PR TITLE
Adding Itau funding

### PR DESCRIPTION
**Pull request by shivapattada**

### CODE REVIEW

File: src/button/component.jsx [hunk 1]
```diff
@@ -532,6 +535,16 @@ export const Button : Component<ButtonOptions> = create({
                     }
                 }
 
+                if (getDomainSetting('disable_itau')) {
```
Comment (bluepnume) [inline src/button/component.jsx]: We should remove this for now -- unless there's a very good reason to add an itau blacklist.

File: src/button/config.js [hunk 2]
```diff
@@ -241,6 +241,40 @@ export const BUTTON_CONFIG : ButtonConfig = {
...
+
+        logoColors:  {
+            [ BUTTON_COLOR.BLUE ]:   BUTTON_LOGO_COLOR.WHITE,
+            [ BUTTON_COLOR.SILVER ]: BUTTON_LOGO_COLOR.BLUE,
+            [ BUTTON_COLOR.BLACK ]:  BUTTON_LOGO_COLOR.WHITE,
+            [ BUTTON_COLOR.WHITE ]:  BUTTON_LOGO_COLOR.BLUE
+        },
+
+        secondaryColors: {
+            [ BUTTON_COLOR.GOLD ]:       BUTTON_COLOR.BLUE,
```
Comment (bluepnume) [inline src/button/config.js]: This should be silver

File: src/button/config.js [hunk 3]
```diff
@@ -241,6 +241,40 @@ export const BUTTON_CONFIG : ButtonConfig = {
...
+            [ BUTTON_COLOR.BLUE ]:       BUTTON_COLOR.SILVER,
+            [ BUTTON_COLOR.SILVER ]:     BUTTON_COLOR.BLUE,
```
Comment (bluepnume) [inline src/button/config.js]: This should be silver

File: src/resources/fundingLogos/index.js [hunk 4]
```diff
@@ -16,6 +16,7 @@ import pp_black     from './pp_black.svg';
 import { paypalLogo } from './paypal';
 import { creditLogo } from './credit';
 import venmo_white  from './venmo_white.svg';
+import itau  from './itau_horizontal.svg';
```
Comment (bluepnume) [inline src/resources/fundingLogos/index.js]: This should just be `itau.svg`

**Comment by shivapattada** [comment]
retest this please

**Comment by shivapattada** [comment]
retest this please

File: src/resources/fundingLogos/itau_vertical.svg [hunk 5]
```diff
@@ -0,0 +1,34 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 350 105">
```
Comment (bluepnume) [inline src/resources/fundingLogos/itau_vertical.svg]: Can we please just use a single SVG file, which works for both horizontal and vertical?

File: src/resources/fundingLogos/index.js [hunk 6]
```diff
@@ -36,6 +37,10 @@ export const fundingLogos = {
         [ BUTTON_LOGO_COLOR.WHITE ]: venmo_white,
         [ BUTTON_LOGO_COLOR.BLUE ]:  venmo_blue
     },
+    [ BUTTON_LOGO.ITAU ]: {
+        [ BUTTON_LOGO_COLOR.WHITE ]: itau,
```
Comment (bluepnume) [inline src/resources/fundingLogos/index.js]: If they're the same color, then let's pick either white or blue.

File: src/button/config.js [hunk 7]
```diff
@@ -241,6 +241,40 @@ export const BUTTON_CONFIG : ButtonConfig = {
...
+
+        colors: [
+            BUTTON_COLOR.BLUE,
+            BUTTON_COLOR.SILVER,
+            BUTTON_COLOR.BLACK,
+            BUTTON_COLOR.WHITE
+        ],
+
+        logoColors:  {
+            [ BUTTON_COLOR.BLUE ]:   BUTTON_LOGO_COLOR.WHITE,
```
Comment (bluepnume) [inline src/button/config.js]: If they're the same color, then let's pick either white or blue and change this config to match.

**Pull request merged**